### DISCUSSION TITLE
Allow sorting shipping methods and payment gateways

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/commerce#2451] [Ideas]
It would be useful if admin could sort (drag and drop) shipping methods and payment gateways, so their order could be changed that way on frontend too.

**Comment by bartrylant** [comment]
I see this FR is 3 years old. It would still come in handy, since you can also set the first shipping method to be the default method (and already be selected on the front-end). However, that's quite useless if you cannot change the order of the shipping methods.

**Comment by wmdhosting** [comment]
+1

**Comment by lukeholder** [comment]
Thanks guys its on the list but not high priority at the moment. Will keep this discussion open!

**Comment by lukeholder** [comment]
Gateway ordering support was adding in Commerce 5. We will look to add shipping method ordering in Commerce 5.5, thanks!
